apps/koala-frontend/src/app/features/sessions/pages/session/session.page.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ApolloTestingController, ApolloTestingModule } from 'apollo-angular/testing';
+import { SessionsModule } from '../../sessions.module';
+
+import { SessionPage } from './session.page';
+
+describe('SessionCreateComponent', () => {
```

Comment:
I guess we should this file as we did for the other components as long as we do not want to add component tests (which I cannot see here at the moment)